templates/definition/meter/enphase.yaml

```diff
@@ -41,7 +41,7 @@ render: |
       password: {{ .token }}
     insecure: true
     {{- end }}
-    jq: if .production[] | select(.measurementType == "production").activeCount >= 1 then .production[] | select(.measurementType == "production").wNow else .production[] | select(.type == "inverters").wNow end
+    jq: if (.production | length) > 1 and (.production[] | select(.measurementType == "production").activeCount >= 1) then .production[] | select(.measurementType == "production").wNow else .production[] | select(.type == "inverters").wNow end
```

Comment:
>1 oder 0?

Reply:
It's 1. If meters are not enabled/available, length of production is 1 (inverters). Otherwise it's 2:

ivoks@rpi:/tmp$ cat sample1.js | jq '.production | length'
1
ivoks@rpi:/tmp$ cat sample1.js 
{"production":[{"type":"inverters","activeCount":22,"readingTime":1734687932,"wNow":442,"whLifetime":6788138}],"storage":[{"type":"acb","activeCount":0,"readingTime":0,"wNow":0,"whNow":0,"state":"idle"}]}

ivoks@rpi:/tmp$ cat sample2.js | jq '.production | length'
2
ivoks@rpi:/tmp$ cat sample2.js 
{"production":[{"type":"inverters","activeCount":20,"readingTime":0,"wNow":123,"whLifetime":35139},{"type":"eim","activeCount":0,"measurementType":"production","readingTime":1735189453,"wNow":-1.385,"whLifetime":0.015,"varhLeadLifetime":167.069,"varhLagLifetime":500.437,"vahLifetime":94240.422,"rmsCurrent":0.364,"rmsVoltage":681.112,"reactPwr":0.552,"apprntPwr":82.03,"pwrFactor":0.0,"whToday":0.015,"whLastSevenDays":0.015,"vahToday":94240.422,"varhLeadToday":167.069,"varhLagToday":500.437}],"consumption":[{"type":"eim","activeCount":0,"measurementType":"total-consumption","readingTime":1735189453,"wNow":-0.381,"whLifetime":2183.108,"varhLeadLifetime":413.248,"varhLagLifetime":1357.325,"vahLifetime":11417.9,"rmsCurrent":0.485,"rmsVoltage":681.107,"reactPwr":-1.254,"apprntPwr":330.597,"pwrFactor":-0.0,"whToday":2183.108,"whLastSevenDays":2183.108,"vahToday":11417.9,"varhLeadToday":413.248,"varhLagToday":1357.325},{"type":"eim","activeCount":0,"measurementType":"net-consumption","readingTime":1735189453,"wNow":1.004,"whLifetime":2185.268,"varhLeadLifetime":246.179,"varhLagLifetime":856.888,"vahLifetime":11417.9,"rmsCurrent":0.121,"rmsVoltage":681.102,"reactPwr":-0.702,"apprntPwr":27.822,"pwrFactor":0.0,"whToday":0,"whLastSevenDays":0,"vahToday":0,"varhLeadToday":0,"varhLagToday":0}],"storage":[{"type":"acb","activeCount":0,"readingTime":0,"wNow":0,"whNow":0,"state":"idle"}]}